### DISCUSSION TITLE
The settings of all game options should be saved in the configuration file "fheroes2.cfg"

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2732] [Ideas]
The settings of all game options should be saved in the configuration file "fheroes2.cfg" so that when the user installs a new version of the game, and uses this "fheroes2.cfg" file, he doesn't have to set all options again according to his wishes.

**Comment by Branikolog** [comment]
I believe, up to final release we won't have so much options there. :)
So, I'm not sure we need this "save". )

**Comment by LeHerosInconnu** [comment]
This options window has been removed from fheroes2.
I close the discussion.
